src/ralph/back_office/models.py

```diff
@@ -50,6 +50,7 @@
 class Warehouse(NamedMixin, TimeStampMixin, models.Model):
     _allow_in_dashboard = True
     stocktaking_enabled = models.BooleanField(default=False)
+    stocktaking_tag_suffix = models.CharField(max_length=8, default='WHOUSE')
```

Comment:
why not empty default? i'd check if `stocktaking_tag_suffix` is set and only if it is i'd set this tag.